rpe/resources/gcp.py

```diff
@@ -431,3 +432,17 @@ def _update_request_args(self, body):
                 'updateMask': "bindings,etag,auditConfigs"
             }
         }
+
+class GcpProjectService(GoogleAPIResource):
+
+    service_name = "serviceusage"
+    resource_path = "services"
+    version = "v1"
+
```

Comment:
Since there is no update method, can we override the update method in this class and have it raise a NotImplementedError?

Reply:
Is that really necessary?  If something tried to update, the API should return an error that can't be retried.

Reply:
The update function will call self._update_request_args, and fail because its not defined for this class. So the exception that ends up in the log message will be confusing

Reply:
oh I see, that makes sense -  @jacquelinekenny could you make this change?

Reply:
Added update function which raises NotImplemented exception